src/auth/auth.module.ts

```diff
@@ -0,0 +1,27 @@
+import { Module } from '@nestjs/common';
+import { JwtModule, JwtService } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+import { AuthController } from './auth.controller';
+import { RedisModule } from '@liaoliaots/nestjs-redis';
```

Comment:
PullProofAI

There is a missing import statement for the `Env` class.
```typescript
+ import { Env } from 'src/env';
```



<!-- This is an auto-generated comment by PullProofAI -->